src/lwc/geFormWidgetTokenizeCard/geFormWidgetTokenizeCard.js

```diff
@@ -108,119 +132,81 @@ export default class geFormWidgetTokenizeCard extends LightningElement {
-    }
-
-    /*******************************************************************************
-    * @description Method listens for a message from the visualforce iframe.
-    * Rejects any messages from an unknown origin.
-    */
-    registerPostMessageListener() {
-        let component = this;
-        window.onmessage = async function (event) {
-            component.visualforceOrigin = component._visualforceOriginUrls.find(
-                origin => event.origin === origin.value).value;
-            if (component.visualforceOrigin) {
-                const message = JSON.parse(event.data);
-                component.handleMessage(message);
-            }
-        }
-    }
-
-    /*******************************************************************************
+    /***
     * @description Method handles messages received from iframed visualforce page.
     *
     * @param {object} message: Message received from iframe
     */
     async handleMessage(message) {
-        if (message.error || message.token) {
-            if (isFunction(this.tokenCallback)) {
-                this.tokenCallback(message);
-            }
-        } else if (message.isLoaded) {
+        tokenHandler.handleMessage(message);
+
+        if (message.isLoaded) {
             this.isLoading = false;
         }
     }
 
-    /*******************************************************************************
+    /***
     * @description Method sends a message to the visualforce page iframe requesting
     * a token. Response for this request is found and handled in
     * registerPostMessageListener.
     */
     requestToken() {
-        const iframe = this.template.querySelector(`[data-id='${this.CUSTOM_LABELS.commonPaymentServices}']`);
-
-        if (iframe) {
-            const tokenPromise = new Promise((resolve, reject) => {
+        this.clearError();
 
-                const timer = setTimeout(() => reject(this.handleTokenizationTimeout()), TOKENIZE_TIMEOUT);
-
-                this.tokenCallback = message => {
-                    clearTimeout(timer);
-                    this.alert = {};
-                    if (message.error) {
-                        reject(this.handleTokenizationError(message));
-                    } else if (message.token) {
-                        resolve({
-                            [DATA_IMPORT_PAYMENT_AUTHORIZATION_TOKEN_FIELD.fieldApiName]: message.token,
-                            [DATA_IMPORT_PAYMENT_STATUS_FIELD.fieldApiName]: this.PAYMENT_TRANSACTION_STATUS_ENUM.PENDING
-                        });
-                    }
-                };
+        const iframe = this.template.querySelector(`[data-id='${this.CUSTOM_LABELS.commonPaymentServices}']`);
 
-            });
+        //The cardholder name is always empty for the purchase Payments Services card tokenization iframe
+        //even though when it is accessible by the Gift Entyr form for Donor Type = Contact.
+        const nameOnCard = null;
```

Comment:
@ptewson-sfdo , @psadie-sfdo , @kenlewis-sfdo , @deepmalamehta :
The GE cardholder name will always be a default. The change in behaviour is to use different constant in the `GE_TokenizeCard` VF page, ie  replace `test name` with `[Not Provided]`. Please let me know if there are any questions or additional change should be done.

Reply:
That's accurate as far as I know - we don't plan on supplying a value there until the gateway metadata endpoint is available to get more accurate information about what additional fields are expected by the gateway for verification, including cardholder name.